app/forms/provide_mobile_number_form.rb

```diff
@@ -0,0 +1,18 @@
+class ProvideMobileNumberForm < Form
+  attribute :provide_mobile_number, :boolean
+
+  validates :provide_mobile_number,
+    inclusion: {
+      in: [true, false],
+      message: "Select yes if you would like to provide your mobile number"
+    },
+    if: -> { claim.has_ecp_or_lupp_policy? }
```

Comment:
```suggestion
    }
```
Not a blocker but I think we can lose this as all it does is allow the user on the student loans journey to submit the form without selecting `true` or `false` and it couples knowledge of a specific journey in a shared form.